src/factories/artifactCluster.ts

```diff
@@ -132,11 +166,18 @@ export async function artifactClusterFactory() {
       return
     }
 
-    const xPos = scale(currentDate) * viewport.scale._x + viewport.worldTransform.tx
-    const x = xPos - element.width / 2
-    const y = application.screen.height - element.height - DEFAULT_ROOT_ARTIFACT_BOTTOM_OFFSET
+    let selectedOffset = 0
```

Comment:
Here, we're doing some fu to place the cluster consistently, accounting for whether it's selected and the blue border is visible or not.

We didn't have to worry about this with nodes on the stage because their position isn't updated constantly like it is at this level.

Reply:
Seems like we could add the border to a different container or use a negative position rather than offsetting the entire element.

Reply:
I can offset the top and left to be a negative value outside the container, but unfortunately the right and bottom are assigned by width and height, so they still increase the container.

Appending the selected outline to a different container would have the drawback of needing to manage it's position and size everywhere it might be used separate from what it's surrounding.

I have updated this code so this offset accounting is only happening in one place. I could see a future where we have one focus factory that finds the selected element and surrounds it independently from all the individual implementations. I'll note that as a stretch goal for now though.